NEXT_CHANGELOG.md

```diff
@@ -127,6 +127,25 @@ There is a new metric called `apollo_router_processing_time` that measures the t
 
 By [@Geal](https://github.com/geal) in https://github.com/apollographql/router/pull/2371
 
+### APQ support in subgraph requests ([PR #2284](https://github.com/apollographql/router/pull/2284))
+
+Automatic persisted queries can now be used for subgraph requests. It is disabled by default, and can be configured for all subgraphs or per subgraph:
+
```

Comment:
```suggestion
### Automated persisted query support for subgraph requests ([PR #2284](https://github.com/apollographql/router/pull/2284))

Automatic persisted queries (APQ) (See useful context [in our Apollo Server docs](https://www.apollographql.com/docs/apollo-server/performance/apq/)) can now be used for subgraph requests. It is disabled by default, and can be configured for all subgraphs or per subgraph:

```

Small preference to avoid abbreviations in titles when they're not necessarily obvious to everyone reading them or understanding their value (or even what they are).   Also, link to more context.

Reply:
:+1: